### PR TITLE
Add Katana Docs

### PR DESCRIPTION
**Pull request by Bisht13**

### CODE REVIEW

File: docs/content/Architecture/design.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The teams are give SSH access to the team pod. Each teams is given a user-password pair that can be used to SSH into the team pod. The team pod is given a public IP address, which can be used to SSH into the team pod from outside of the cluster.
+
+Challenges are pods that are deployed into the team's namespace. On patching, the pod is redeployed.
+
+Katana has its own namespace. This namespace is used to deploy Katana components. These components include flag handler service, challenge checker service, logging service, git server. We will discuss these components in more detail in the next section.
+
+{{< img name="arch" size="large" lazy=false >}} // Not working
```
Comment (Bisht13) [inline docs/content/Architecture/design.md:19]: Check this

File: docs/content/ChallengeChecker/main.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Challenge Checker"
+---
+
+# WIP
```
Comment (Bisht13) [inline docs/content/ChallengeChecker/main.md:5]: @h4shk4t is this finished?

File: docs/content/Katana/setup.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Setup"
+---
+
+Anshul
```
Comment (Bisht13) [inline docs/content/Katana/setup.md:5]: @Perseus-Jackson477 can you pick this up? else ask someone who can finish this.

File: docs/content/KatanaD/main.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: "KatanaD"
```
Comment (Bisht13) [inline docs/content/KatanaD/main.md:2]: Need to update and finish this.

File: docs/content/KatanaD/v1.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+---
+title: "V1 - Challenge Containers"
```
Comment (Bisht13) [inline docs/content/KatanaD/v1.md:2]: Is this finished?

File: docs/content/KatanaD/v2.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+title: "V2 - Challenge Containers"
```
Comment (Bisht13) [inline docs/content/KatanaD/v2.md:2]: Need to finish

File: docs/content/Patching/main.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Patching Service"
```
Comment (Bisht13) [inline docs/content/Patching/main.md:2]: Need to finish this.

File: docs/content/Services/ssh.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+title: "SSH Service"
```
Comment (Bisht13) [inline docs/content/Services/ssh.md:2]: Is this complete? @Perseus-Jackson477

File: docs/content/TeamNamespaces/main.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Team Namespaces"
```
Comment (Bisht13) [inline docs/content/TeamNamespaces/main.md:2]: Is this complete? @h4shk4t

File: docs/content/Architecture/design.md (hunk 1 above)
Comment (ashpect) [reply docs/content/Architecture/design.md:19]: Some issue with svg. Updated it to png.

File: docs/content/TeamNamespaces/main.md (hunk 9 above)
Comment (h4shk4t) [reply docs/content/TeamNamespaces/main.md:2]: Yes

**Pull request merged**